x/qgb/keeper/keeper_data_commitment.go

```diff
@@ -44,14 +44,14 @@ func (k Keeper) GetDataCommitmentForHeight(ctx sdk.Context, height uint64) (type
 		)
 	}
 	latestNonce := k.GetLatestAttestationNonce(ctx)
-	for i := uint64(0); i < latestNonce; i++ {
+	for i := latestNonce; i >= 0; i-- {
```

Comment:
hold off on merging this. golangci-lint made me realize this may loop indefinitely because uint underflow